src/main/java/org/recap/repository/jpa/BibliographicDetailsRepository.java

```diff
@@ -64,23 +62,15 @@
      * @return the bibliographic entity
      */
     BibliographicEntity findByOwningInstitutionIdAndOwningInstitutionBibIdAndIsDeletedFalse(Integer owningInstitutionId, String owningInstitutionBibId);
```

Comment:
We try to use the specification, to avoid the repository explosions. I will write a sample send you back, we incorporate that, right now i will approve it.